modules/Microsoft.VirtualMachineImages/imageTemplates/.test/common/dependencies.bicep

```diff
@@ -58,6 +61,30 @@ resource msi_roleAssignment 'Microsoft.Authorization/roleAssignments@2022-04-01'
   }
 }
 
+resource virtualNetwork 'Microsoft.Network/virtualNetworks@2022-01-01' = {
+  name: virtualNetworkName
+  location: location
+  properties: {
+    addressSpace: {
+      addressPrefixes: [
+        '10.0.0.0/24'
```

Comment:
Just a small change for alignment with the latest changes we did to other other VNET dependencies:

Please introduce a variable `var addressPrefix = '10.0.0.0/24'` and use it for both the VNET and subnet :)